c2rust-analyze/src/context.rs

```diff
@@ -55,10 +58,18 @@ pub use crate::pointer_id::PointerId;
 pub type LTy<'tcx> = LabeledTy<'tcx, PointerId>;
 pub type LTyCtxt<'tcx> = LabeledTyCtxt<'tcx, PointerId>;
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub struct LFnSig<'tcx> {
```

Comment:
Just `L` for `Labeled` seems a bit short here.

Reply:
`LTy` is used pervasively throughout this codebase, so I don't think it needs a long name.  And `LFnSig` is named by analogy with rustc's ordinary `FnSig` and `Ty`.

Reply:
Ok, I think it's fine as long as `L` only means `Labeled`, but if it means `locals` or anything else elsewhere, then it gets very confusing.